packages/vendure-plugin-customer-managed-groups/src/api-extensions.ts

```diff
@@ -61,6 +70,17 @@ export const shopSchema = gql`
     removeCustomerFromMyCustomerManagedGroup(
       customerId: ID!
     ): CustomerManagedGroup!
+
+    """
+    Update a member of one's group
+    """
+    updateCustomerManagedGroupMember(
+      input: UpdateCustomerManagedGroupMemberInput!
+    ): CustomerManagedGroup
+    makeCustomerAdminOfCustomerManagedGroup(
+      groupId: ID!
+      customerId: ID!
+    ): CustomerManagedGroup
```

Comment:
This should be mandatory, I think. We will throw an error if you're not in a group or not allowed to do this. 


```suggestion
    ): CustomerManagedGroup!
```